startLuminoWithRIFComms.sh

```diff
@@ -0,0 +1,130 @@
+
+## Here some explanation of the parameters:
+
+## TOKEN_NETWORK_REGISTRY: The token network registry contract address to be used by the node.
+## SECRET_REGISTRY: The secret registry contract address to be used by the node.
+## ENDPOINT_REGISTRY: The endpoint registry contract address to be used by the node.
+## HUB_MODE: If set, the node will work as a hub
+## RSK_ENDPOINT: The RSK node endpoint URL
+## LUMINO_API_ENDPOINT: The expose url to use by the lumino node
+## NETWORK_ID: the blockchain network ID to connect to
+## RIF_COMMS_NODE_CONFIGURATION: config file of the rif comms node
+## RIF_COMMS_NODE_PORT: port in which rif comms node is going to start
+
+
+# The output log for rif comms node is on rif-communications-pubsub-bootnode under the commsNodeLog.txt name
+
+ARGUMENT_LIST=(
+    "RIF_COMMS_NODE_DIR"
+    "KEYSTORE_PATH"
+    "TOKEN_NETWORK_REGISTRY"
+    "SECRET_REGISTRY"
+    "ENDPOINT_REGISTRY"
+    "HUB_MODE"
+    "RSK_ENDPOINT"
+    "LUMINO_API_ENDPOINT"
+    "NETWORK_ID"
+    "RIF_COMMS_NODE_CONFIGURATION"
+    "RIF_COMMS_NODE_PORT"
+)
+
+
+# read arguments
+opts=$(getopt \
+    --longoptions "$(printf "%s:," "${ARGUMENT_LIST[@]}")" \
+    --name "$(basename "$0")" \
+    --options "" \
+    -- "$@"
+)
+
+eval set --$opts
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --RIF_COMMS_NODE_DIR)
+            RIF_COMMS_NODE_DIR=$2
+            shift 2
+            ;;
+        --KEYSTORE_PATH)
+            KEYSTORE_PATH=$2
+            shift 2
+            ;;
+        --TOKEN_NETWORK_REGISTRY)
+            TOKEN_NETWORK_REGISTRY=$2
+            shift 2
+            ;;
+        --SECRET_REGISTRY)
+            SECRET_REGISTRY=$2
+            shift 2
+            ;;
+        --ENDPOINT_REGISTRY)
+            ENDPOINT_REGISTRY=$2
+            shift 2
+            ;;
+        --HUB_MODE)
+            HUB_MODE=$2
+            shift 2
+            ;;
+        --RSK_ENDPOINT)
+            RSK_ENDPOINT=$2
+            shift 2
+            ;;
+        --LUMINO_API_ENDPOINT)
+            LUMINO_API_ENDPOINT=$2
+            shift 2
+            ;;
+        --NETWORK_ID)
+            NETWORK_ID=$2
+            shift 2
+            ;;
+        --RIF_COMMS_NODE_CONFIGURATION)
+            RIF_COMMS_NODE_CONFIGURATION=$2
+            shift 2
+            ;;
+        --RIF_COMMS_NODE_PORT)
+            RIF_COMMS_NODE_PORT=$2
+            shift 2
+            ;;
+        *)
+            break
+            ;;
+    esac
+done
+
+# Start rif comms node
+export NODE_ENV=$RIF_COMMS_NODE_CONFIGURATION
```

Comment:
As the comment above you can just validate the rif comms params and then send all the other params to startLumino since that script already validate the others.